ROADMAP.md

```diff
@@ -2,60 +2,47 @@
 
 ## About this Document
 
-This document provides an overview of the major themes driving Tanzu Community Edition development, as well as some of the features and
-capabilities that will manifest those priorities, grouped into 0-3 month, 3-6 month, and future buckets. Most items are gathered from
-the community or include a feedback loop with the community. This should serve as a reference point for Tanzu Community Edition users
-to help us prioritize and to supply input on unmet needs, and contributors to understand where the project is heading, and help
-contributors to determine if a contribution could be conflicting with a longer term plan.
+This document provides an overview of the major themes driving Tanzu Community Edition development, as well as constituent features and capabilities. Roadmap items are tracked by their implementation state within the enhancements proposal process: Awaiting Proposal, Proposal in Review, and Implementation in Progress. Most items are gathered from the community or include a feedback loop with the community.
 
-More detail about short-term delivery plans can be found in the [milestones](https://github.com/vmware-tanzu/community-edition/milestones) in
-the [GitHub project repo](https://github.com/vmware-tanzu/community-edition).
+This roadmap should serve as a reference point for Tanzu Community Edition users to help us prioritize existing features and provide input on unmet needs, and for contributors to understand where the project is heading. A roadmap also helps our contributor community to determine if a proposed enhancement conflicts with the long-term plan.
 
-## How to help
+Enhancement states help everyone understand if something is in the early stages of an idea or if it has wider community buy-in and committed resources. This is important because it helps limit waste, and focus work on the most important and timely activities. The stages are described below:
 
-Discussion on the roadmap can take place in threads under [Issues](https://github.com/vmware-tanzu/community-edition/issues)
-or in [community meetings, our Google Group, or our Slack channel](https://tanzucommunityedition.io/community/).
-Please open and comment on an issue if you want to provide suggestions and feedback to an item in the roadmap.
-Please review the roadmap to avoid potential duplicated effort.
+.Awaiting Proposal - these enhancements are in the idea mode, and the project maintainers are awaiting a detailed proposal. This state is useful to explore different solutions to a known problem and encourage discussion.
```

Comment:
What is this `.` notation? Also these are are being formatted as a single paragraph. Did you mean to make this a bulleted list?

You can check your formatting by viewing the file in GitHub.

![image](https://user-images.githubusercontent.com/6200057/151604359-251f07db-856a-4239-9831-211094d74096.png)

Reply:
Yes, it came from a "how to MD" doc but apparently an incorrect one...

Reply:
Looks great now, thanks for resolving this.